### PR TITLE
[IOSP-607] Update PullAssigners document to remove reference to Pull Panda and explain GitHub's code review assignment

### PR DESCRIPTION
**Pull request by j531**
We no longer user Pull Panda to assign reviews. This PR updates the documentation to remove references to Pull Panda and adds a section about how to use GitHub's code review assignment in its place.

### CODE REVIEW

File: Cookbook/Technical-Documents/PullAssigners.md
Comment (sdaniel55) [inline Cookbook/Technical-Documents/PullAssigners.md]: We should probably share this in `#ios-pedia`

**Review by danspndl — APPROVED** [review]
Looks good, thanks! 👏

File: Cookbook/Technical-Documents/PullAssigners.md
Comment (konrad-em) [reply Cookbook/Technical-Documents/PullAssigners.md]: Good point! It could be also updated [here](https://github.com/babylonhealth/ios-playbook/blob/master/Cookbook/Technical-Documents/OutOffOfficeRequest.md)

File: Cookbook/Technical-Documents/PullAssigners.md
Comment (konrad-em) [inline Cookbook/Technical-Documents/PullAssigners.md]:
```suggestion
**Note:** You'll probably want to tick "If assigning team members, don't notify the entire team" to prevent the entire team from getting notifications for every PR, even if they haven't been individually assigned to it.
```

File: Cookbook/Technical-Documents/PullAssigners.md
Comment (konrad-em) [inline Cookbook/Technical-Documents/PullAssigners.md]:
```suggestion
1. First, you need a GitHub team containing the list of all the people you want GitHub to pick reviewers from.
```

**Review by konrad-em — APPROVED** [review]

**Review by ilyapuchka — APPROVED** [review]

**Comment by j531** [comment]
Thanks for reviews @sdaniel55, @konrad-em, @ilyapuchka 👍

I've updated the Out of Office doc and I'll merge at lunch time if there are no further suggestions.

File: Cookbook/Technical-Documents/PullAssigners.md
Comment (AliSoftware) [inline Cookbook/Technical-Documents/PullAssigners.md]: My immediate question when reading this was "Where is that N configured?". Then continuing reading on I saw the section further below explaining it.
So might be a good idea to link to the instructions below from here

```suggestion
- GitHub will then pick N reviewers from that team ([see below how to configure that number N](#enabling-github-s-code-review-assignment)) and unassign the team itself.
```

File: Cookbook/Technical-Documents/PullAssigners.md
Comment (AliSoftware) [inline Cookbook/Technical-Documents/PullAssigners.md]: This comment in the `CODEOWNERS` file should be removed as it mentions PullAssigner (and we don't use the concept of a proxy team anymore with GitHub method anyway)

You might want to update the comment to something like this instead
```suggestion
# Global rule for the whole codebase.
# GitHub's Code Reviewer assignment feature will only pick N people from that list
#   to assign them as reviewers. To configure this, go to your Team's page
#   (https://github.com/orgs/babylonhealth/teams/<yourteam>) then to
#   Settings > Code review assignment
```

File: Etiquette/CODE_REVIEW.md
Comment (AliSoftware) [inline Etiquette/CODE_REVIEW.md]: Is it really random though? Might be interesting (and a bit reassuring) to mention that it's more like round-robin to ensure fair balance

**Review by AliSoftware — COMMENTED** [review]
Great doc update 👍

Just commented with a couple of nits. But I also realized that we don't explicitly document in the Onboarding docs that new hires should ask to be added to the `ios-devs` team. It would be good to include that as part of this PR too, since being added to the team impacts how code reviewers assignment works (and we've seen in the past that not everybody is added to that team since currently we have 2 different teams that provide access to our repos and some people are on one but not the other)

This will require updating at least both [NewHireChecklist.md](https://github.com/babylonhealth/ios-playbook/blob/master/Cookbook/Technical-Documents/NewHiresCheckList.md#github-access) and the line about GitHub in the table in [ToolsAndServices](https://github.com/babylonhealth/ios-playbook/blob/master/Cookbook/Technical-Documents/ToolsAndServices.md)

File: Cookbook/Technical-Documents/PullAssigners.md
Comment (AliSoftware) [inline Cookbook/Technical-Documents/PullAssigners.md]: Since this level-2 section doesn't have more than one level-3 subsection anymore (now that you deleted all the other subsections that are not needed anymore), this title hierarchy is a bit redundant

```suggestion
## Configure a GitHub Team containing the reviewers to pick from
```

File: Cookbook/Technical-Documents/PullAssigners.md
Comment (j531) [reply Cookbook/Technical-Documents/PullAssigners.md]: Oops, missed this one (GitHub's search is terrible). Thanks for spotting!

File: Cookbook/Technical-Documents/PullAssigners.md
Comment (j531) [reply Cookbook/Technical-Documents/PullAssigners.md]: Good suggestion 👍 The link doesn't seem to be working in the file viewer so I'll check it when it's merged.

File: Etiquette/CODE_REVIEW.md
Comment (j531) [reply Etiquette/CODE_REVIEW.md]: I've updated this to mention the routing algo. Let me know if it makes sense.

File: Cookbook/Technical-Documents/PullAssigners.md
Comment (AliSoftware) [reply Cookbook/Technical-Documents/PullAssigners.md]: Checked the anchor via Google Chrome inspected while in file viewer, the proper link should be this instead: `#enabling-githubs-code-review-assignment`

File: Cookbook/Technical-Documents/PullAssigners.md
Comment (AliSoftware) [inline Cookbook/Technical-Documents/PullAssigners.md]:
```suggestion
- GitHub will then pick N reviewers from that team ([see below how to configure that number N](#enabling-githubs-code-review-assignment)) and unassign the team itself.
```

**Comment by j531** [comment]
@AliSoftware 

Included your suggestions and updated NewHireChecklist / ToolsAndServices.

File: Cookbook/Technical-Documents/NewHiresCheckList.md
Comment (AliSoftware) [inline Cookbook/Technical-Documents/NewHiresCheckList.md]: 👌

**Review by AliSoftware — APPROVED** [review]

**Pull request merged**
